zebra-chain/src/parameters/network.rs

```diff
@@ -108,17 +164,32 @@ impl fmt::Display for Network {
 }
 
 impl Network {
+    /// Creates a new [`Network::Testnet`] with the default Testnet network parameters.
+    pub fn new_default_testnet() -> Self {
+        Self::Testnet(Arc::new(NetworkParameters::default()))
+    }
+
+    /// Returns true if the network is the default Testnet, or false otherwise.
+    pub fn is_default_testnet(&self) -> bool {
+        if let Self::Testnet(params) = self {
+            params.is_default_testnet()
+        } else {
+            false
+        }
+    }
+
     /// Returns an iterator over [`Network`] variants.
     pub fn iter() -> impl Iterator<Item = Self> {
         // TODO: Use default values of `Testnet` variant when adding fields for #7845.
-        [Self::Mainnet, Self::Testnet].into_iter()
+        [Self::Mainnet, Self::new_default_testnet()].into_iter()
     }
 
     /// Get the default port associated to this network.
     pub fn default_port(&self) -> u16 {
         match self {
             Network::Mainnet => 8233,
-            Network::Testnet => 18233,
+            // TODO: Add a `default_port` field to `NetworkParameters` to return here.
```

Comment:
In zcashd, Regtest has its own default port (nDefaultPort = 18344; from zcash/src/chainparams.cpp:714). It could be good to add this to the TODO?

Reply:
Added in https://github.com/ZcashFoundation/zebra/pull/8368/commits/dbf32960639e5741bd9f02ec163534de18c45c5b (here: [`zebra_chain::parameters::network:235`](https://github.com/ZcashFoundation/zebra/pull/8368/commits/dbf32960639e5741bd9f02ec163534de18c45c5b#diff-4dcd6cfa50ff2b87d1dc68673b1ec3a39b816a0473d15a17423a80ae80029bcaR235)).